lambda/rotator-snowflake/rotate/rotate.go

```diff
@@ -0,0 +1,169 @@
+package rotate
+
+import (
+	"bytes"
+	"context"
+	"crypto/rsa"
+	"crypto/x509"
+	"database/sql"
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/chanzuckerberg/go-misc/keypair"
+	"github.com/chanzuckerberg/go-misc/lambda/rotator-snowflake/setup"
+	"github.com/chanzuckerberg/go-misc/snowflake"
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
+	"github.com/xinsnake/databricks-sdk-golang/aws"
+	"github.com/xinsnake/databricks-sdk-golang/aws/models"
+)
+
+func getUsers() ([]string, error) {
+	usersList := os.Getenv("CURRENT_USERS") //Comma-delimited users list
+	userSlice := strings.Split(usersList, ",")
+	return userSlice, nil
+}
+
+func buildSnowflakeSecrets(connection *sql.DB, username string, privateKey *bytes.Buffer) (map[string]string, error) {
```

Comment:
```suggestion
func buildSnowflakeSecrets(connection *sql.DB, username string, newPrivateKey *bytes.Buffer) (map[string]string, error) {
```
maybe makes it clearer this is the new key you'll write